### PR TITLE
PROD-222 Add more sentry errors

### PR DESCRIPTION
**Pull request by neel-ds**
- Description
Handle exceptions via Sentry in more actions and upgrade the current exception
- What are the steps to test that this code is working?
Try to break something in login, reveal, burn or claim
- Screen shots or recordings for UI changes
NA

### CODE REVIEW

File: app/actions/answer.ts [hunk 1]
```diff
@@ -72,107 +74,118 @@ export async function answerQuestion(request: SaveQuestionRequest) {
...
+      );
 
-    if (allQuestionOptions.length === allQuestionAnswers.length) {
-      fungibleAssetRevealTasks.push(
+      const fungibleAssetRevealTasks = [
         incrementFungibleAssetBalance({
           asset: FungibleAsset.Point,
-          amount: pointsPerAction[TransactionLogType.AnswerDeck],
-          transactionLogType: TransactionLogType.AnswerDeck,
+          amount: pointsPerAction[TransactionLogType.AnswerQuestion],
```
Comment (marvinmarnold) [inline app/actions/answer.ts:156]: What do these changes have to do with the Linear issue?

**Review by marvinmarnold — CHANGES_REQUESTED** [review]

File: app/actions/answer.ts (hunk 1 above)
Comment (neel-ds) [reply app/actions/answer.ts:156]: I wrapped the code in a try-catch block, which caused some restructuring across different lines. Technically, there are no functional changes, but Git is detecting it as code changes. I compared it line by line. Please ignore it.

**Comment by marvinmarnold** [comment]
@neel-ds you will follow up with additional commits, right?

**Comment by neel-ds** [comment]
> @neel-ds you will follow up with additional commits, right?

Yes!

File: app/hooks/useReveal.ts [hunk 2]
```diff
@@ -301,6 +305,12 @@ export function useReveal({ wallet, address, bonkBalance }: UseRevealProps) {
           errorToast(
             "Error while confirming transaction. Bonk was not burned. Try again.",
           );
+          class BurnError extends Error {}
+          const burnError = new BurnError(
+            `User with id: ${payload?.sub} is having trouble burning questions with ids: ${revealQuestionIds}`,
+            { cause: res.value.err },
+          );
+          Sentry.captureException(burnError);
```
[the remark this answers is not in the document]
Comment (neel-ds) [reply app/hooks/useReveal.ts:313]: done.

**Review by marvinmarnold — APPROVED** [review]

**Pull request merged**